example/spec/syncGranule/SyncGranuleDuplicateHandlingSkipSpec.js

```diff
@@ -0,0 +1,135 @@
+const fs = require('fs');
+const path = require('path');
+const {
+  aws: {
+    headObject,
+    parseS3Uri,
+    s3
+  },
+  stringUtils: { globalReplace },
+  testUtils: {
+    randomString
+  }
+} = require('@cumulus/common');
+const { api: apiTestUtils, buildAndExecuteWorkflow, LambdaStep } = require('@cumulus/integration-tests');
+const {
+  deleteFolder,
+  loadConfig,
+  templateFile,
+  timestampedTestDataPrefix,
+  uploadTestDataToBucket
+} = require('../helpers/testUtils');
+const {
+  loadFileWithUpdatedGranuleId,
+  setupTestGranuleForIngest
+} = require('../helpers/granuleUtils');
+const config = loadConfig();
+const lambdaStep = new LambdaStep();
+const workflowName = 'SyncGranuleDuplicateSkipTest';
+
+const granuleRegex = '^MOD09GQ\\.A[\\d]{7}\\.[\\w]{6}\\.006\\.[\\d]{13}$';
+const testDataGranuleId = 'MOD09GQ.A2016358.h13v04.006.2016360104606';
+
+const outputPayloadTemplateFilename = './spec/syncGranule/SyncGranule.output.payload.template.json';
+const templatedOutputPayloadFilename = templateFile({
+  inputTemplateFilename: outputPayloadTemplateFilename,
+  config: config.SyncGranule
+});
+
+const s3data = [
+  '@cumulus/test-data/granules/MOD09GQ.A2016358.h13v04.006.2016360104606.hdf.met',
+  '@cumulus/test-data/granules/MOD09GQ.A2016358.h13v04.006.2016360104606.hdf',
+  '@cumulus/test-data/granules/MOD09GQ.A2016358.h13v04.006.2016360104606_ndvi.jpg'
+];
+
+describe('When the Sync Granule workflow is configured to skip new data when encountering duplicate filenames', () => {
+  const testDataFolder = timestampedTestDataPrefix(`${config.stackName}-IngestGranuleSuccess`);
+  const inputPayloadFilename = './spec/syncGranule/SyncGranule.input.payload.json';
+  let inputPayload;
+  let expectedPayload;
+  const collection = { name: 'MOD09GQ', version: '006' };
+  const provider = { id: 's3_provider' };
+  let workflowExecution;
+
+  beforeAll(async () => {
+    // upload test data
+    await uploadTestDataToBucket(config.bucket, s3data, testDataFolder, false);
+
+    const inputPayloadJson = fs.readFileSync(inputPayloadFilename, 'utf8');
+
+    // update test data filepaths
+    const updatedInputPayloadJson = globalReplace(inputPayloadJson, 'cumulus-test-data/pdrs', testDataFolder);
+    inputPayload = await setupTestGranuleForIngest(config.bucket, updatedInputPayloadJson, testDataGranuleId, granuleRegex);
+
+    const granuleId = inputPayload.granules[0].granuleId;
+    expectedPayload = loadFileWithUpdatedGranuleId(templatedOutputPayloadFilename, testDataGranuleId, granuleId);
```

Comment:
This line can be deleted as it is duplicated and overwritten in the next few lines.